docs/install/run-the-kit.md

```diff
@@ -20,7 +20,7 @@ Listening on port 3000 url: http://localhost:3000
 
 ## Check it works
 
-In your web browser, visit [http://localhost:3000](http://localhost:3000)
+In your web browser, visit <a href="http://localhost:3000" target="_blank">http://localhost:3000<span class="visuallyHidden"> (opens in a new window)</span></a>
```

Comment:
ok, but I think the warning is useful to all users, not just screen readers. And should it be "opens in a new tab"?